src/components/textfield/index.tsx

```diff
@@ -156,10 +156,10 @@ const TextField = (props: TextFieldTypeProps) => {
 
         '& .MuiAutocomplete-endAdornment .MuiSvgIcon-root': {
           color: !props.disabled
-            ? (endIcon?.props['color'] ?? 'var(--black)')
+            ? (!!endIcon && endIcon.props && 'color' in (endIcon?.props as { color: string }) ? endIcon.props['color'] : 'var(--black)')
```

Comment:
not a fan of this code but it's the only way that fixed the issue
doing `endIcon?.props?.color || 'var(--black)'` didn't do the job